src/codegen.ts

```diff
@@ -30,33 +29,11 @@ export async function codegen(contractName, abi) {
         '@wharfkit/session'
     )
     const importContractStatement = generateImportStatement(
-        ['Contract', 'Table', 'TableCursor', 'QueryOptions', 'QueryOptions', 'Query'],
+        ['Contract', 'ContractArgs', 'blobStringToAbi'],
         '@wharfkit/contract'
     )
 
-    const {methods: actionMethods, interfaces: actionsInterfaces} = generateActions(
-        contractName,
-        namespaceName,
-        ABI.from(abi)
-    )
-
-    // Generate actions namespace
-    const actionsNamespace = generateNamespace(namespaceName, [
-        generateNamespace('actions', actionMethods),
-    ])
-
-    const tableClasses: ts.ClassDeclaration[] = []
-    const tableInterfaces: ts.InterfaceDeclaration[] = []
-
-    for (const table of abi.tables) {
-        const {classDeclaration} = await generateTableClass(contractName, namespaceName, table, abi)
-        tableClasses.push(classDeclaration)
-    }
-
-    // Generate tables namespace
-    const tableNamespace = generateNamespace(namespaceName, [
-        generateNamespace('tables', tableClasses),
-    ])
+    const {classDeclaration} = await generateContractClass(namespaceName, contractName, abi)
```

Comment:
I wonder if it makes sense to put the `Contract` inside of the generated namespace 🤔 

Instead of:

```ts
export class _EosioToken extends Contract {
    ...
}
export namespace _EosioToken {
    export namespace types {
        ...
    }
}
```

We do something like:

```ts
export namespace _EosioToken {
    export class Contract extends BaseContract {
        ...
    }    
    export namespace types {
        ...
    }
}
```